src/main/java/org/cryptomator/common/vaults/VaultListManager.java

```diff
@@ -114,6 +115,15 @@ private Optional<Vault> get(Path vaultPath) {
 	private Vault create(VaultSettings vaultSettings) {
 		var wrapper = new VaultConfigCache(vaultSettings);
 		try {
+			if (Objects.isNull(vaultSettings.lastKnownKeyLoader.get())) {
+				try {
+					var keyIdScheme = wrapper.get().getKeyId().getScheme();
+					vaultSettings.lastKnownKeyLoader.set(keyIdScheme);
+				} catch (NoSuchFileException e) {
+					LOG.error("Configuration file missing.", e);
+					return vaultComponentFactory.create(vaultSettings, wrapper, ERROR, e).vault();
+				}
```

Comment:
This is basically the same as lines 132ff. Is there a reason to keep the special handling of `NoSuchFileException` ?